content/applications/sales/rental.rst

```diff
@@ -2,46 +2,123 @@
 Rental
 ======
 
-**Odoo Rental** is a comprehensive solution to manage your rentals.
+The Odoo *Rental* application provides comprehensive solutions to configure and manage rentals.
 
-From a single view, you can send out quotations, confirm orders, schedule rentals, register when
-products are picked up and returned, and invoice your customers.
+Send quotations, confirm orders, schedule rentals, register products when they are picked up and
+returned, and invoice customers from this single platform.
 
 .. seealso::
    - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
    - `Odoo Tutorials: Rental <https://www.odoo.com/slides/rental-48>`_
 
 .. _rental/pricing:
 
-Rental Pricing
+Dashboard
+=========
+
+When the *Rental* application is opened, Odoo reveals the :guilabel:`Rental Orders` dashboard.
+
+.. image:: rental/rental-orders-dashboard.png
+   :align: center
+   :alt: Example of the Rental Orders dashboard available in the Odoo Rental application.
+
+In the default kanban view, all rentals are visible. Each rental card displays the customer name,
+the price of the rental, the related sales order number, along with the status of the rental.
+
+.. note::
+   Rental kanban cards that do **not** display a rental status means those rentals have confirmed
+   quotations, but haven't been picked up yet.
+
+On the left sidebar, the :guilabel:`Rental Status` for each rental can be found. Beneath that, the
+:guilabel:`Invoice Status` of the rentals is accessible. Clicking any option in the left sidebar
+reveals that designated filter option in the dashboard for easier access.
+
+Settings
+========
+
+To access the settings of the Odoo *Rental* application, navigate to :menuselection:`Rental app -->
```

Comment:
> To access the settings of the Odoo *Rental* application

While this is concise, I think we can elaborate a little more about why the reader might need extra settings in their rental app. For example, 

```suggestion
To configure additional rental delay costs, availability of rental items, or minimum time of rental,
navigate to :menuselection:`Rental app -->
```